src/index.js

```diff
@@ -10,13 +10,13 @@ const moment = require('moment');
 
 
 let selectionBox = document.querySelector('.selection-box');
-let bookTrip = document.querySelector('.book-trip');
+let calculateCost = document.querySelector('.calculate-cost');
 
 let tripsToDisplay, allDestinations, user;
```

Comment:
still don't know if I need trips to display. Prolly not, huh.

Reply:
If it's not called, THROW IT IN THE TRASH!